File: branches/issue/13/tags/source/mergepoint00/DocTestVB/Syntax/TestEnum.vb
Namespace Syntax

	''' <summary>
	''' Enum used in testing syntax generation.
	''' </summary>
	<Flags(), _
	 System.Xml.Serialization.XmlRootAttribute("root", DataType:="string")> _
	Public Enum TestEnum As Long

		''' <summary>
		''' First value.
		''' </summary>
		Value1

		''' <summary>
		''' Second value.
		''' </summary>
		Value2

	End Enum

End Namespace

File: branches/issue/7/dev/DocTestCS/Syntax/TestEnum.cs
using System;
using System.Xml.Serialization;

namespace DocTest.Syntax
{
	/// <summary>
	/// Enum used in testing syntax generation.
	/// </summary>
	[Flags]
	[XmlRoot("root", DataType="string")]
	public enum TestEnum : long
	{
		/// <summary>
		/// First value.
		/// </summary>
		Value1,

		/// <summary>
		/// Second value.
		/// </summary>
		Value2
	}
}
